tests/test_client_credential.py

```diff
@@ -105,6 +105,11 @@ def test_client_credential_with_hashed_client_secret(self):
         response = self.client.post(reverse("oauth2_provider:token"), data=token_request_data, **auth_headers)
         self.assertEqual(response.status_code, 401)
 
+        # sending the hashed secret should return a 401
+        auth_headers = get_basic_auth_header(self.application.client_id, self.application.client_secret)
+        response = self.client.post(reverse("oauth2_provider:token"), data=token_request_data, **auth_headers)
+        self.assertEqual(response.status_code, 401)
```

Comment:
You'll need to fix the code -- I believe the test is correct -- since the code is still returning a 200. I had to enable running tests for you again because you switched to a different github user. 

Consider running tox locally before pushing and requesting a review.

Reply:
hm. I did run the tests before opening the PR. I will close this and open under my other account (apologies -- didn't notice I was in the "wrong" browser window).